guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,14 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+ifndef::foreman-deb[]
 There are no highlights with Foreman {ProjectVersion}.
+endif::[]
+ifdef::foreman-deb[]
+=== Running Foreman on Debian 12 (Bookworm)
```

Comment:
```suggestion
=== Running Foreman on Debian 12
```
I'd drop this detail from the heading because it's repetitive, but it isn't a blocker. Feel free to disagree :)